### PR TITLE
Rebased with upstream 3.3.5, converted to 3.3.6

### PR DESCRIPTION
**Pull request by skateman**
To test this, update the patternfly line in the manageiq gemfile to:
`gem 'patternfly-sass', :github => 'skateman/patternfly-sass', :branch => '3.3.6-tertiary'`

### CODE REVIEW

File: spec/html/vertical-navigation.html [hunk 1]
```diff
@@ -338,7 +338,11 @@
       <div class="row row-cards-pf">
       <!-- Important:  if you need to nest additional .row within a .row.row-cards-pf, do *not* use .row-cards-pf on the nested .row  -->
         <div class="col-xs-12 col-sm-6 col-md-3">
+<<<<<<< f00b739f602221d24a347355a9219afe90726e88
 <div class="card-pf card-pf-accented card-pf-aggregate-status">
+=======
+          <div class="card-pf card-pf-accented card-pf-aggregate-status">
+>>>>>>> Modified vertical navigation templates so they can be included in patternfly-toolkit.
```
Comment (mfalesni) [inline spec/html/vertical-navigation.html]: @skateman rebase artifacts? :stuck_out_tongue:

Comment (skateman) [inline spec/html/vertical-navigation.html]: @jeff-phillips-18 rebase artifacts https://github.com/jeff-phillips-18/patternfly/blob/tertiary-nav/tests/vertical-navigation.html#L341

Comment (jeff-phillips-18) [inline spec/html/vertical-navigation.html]: @skateman I have fixed the tertiary-nav branch.

Comment (skateman) [inline spec/html/vertical-navigation.html]: fixed in -sass too, thanks

File: assets/javascripts/tertiary-nav.js [hunk 2]
```diff
@@ -337,8 +337,8 @@
           });
         });
 
-        var hoverDelay = 500;
-        var hideDelay = hoverDelay + 200;
+        const hoverDelay = 500;
+        const hideDelay = hoverDelay + 200;
```
Comment (himdel) [inline assets/javascripts/tertiary-nav.js]: Broken! ;)

Comment (skateman) [inline assets/javascripts/tertiary-nav.js]: @jeff-phillips-18 https://github.com/jeff-phillips-18/patternfly/blob/tertiary-nav/dist/js/tertiary-nav.js#L340

Comment (jeff-phillips-18) [inline assets/javascripts/tertiary-nav.js]: @skateman @himdel Fixed in the tertiary-nav branch

**Comment by romanblanco** [comment]
LGTM

**Comment by himdel** [comment]
Skipped the examples, but LGTM too..

**Pull request merged**